patterns/behavioral/command.py

```diff
@@ -1,61 +1,71 @@
 """
+Command pattern decouples the object invoking a job from the one who knows
+how to do it. As mentioned in the GoF book, a good example is in menu items.
+You have a menu that has lots of items. Each item is responsible for doing a
+special thing and you want your menu item just call the `execute` method when
+it is pressed. To achieve this you implement a command object with the `execute`
+method for each menu item and pass to it.
+
 *TL;DR
-Encapsulates all information needed to perform an action or trigger an event.
+Object oriented implementation of callback functions.
 
 *Examples in Python ecosystem:
 Django HttpRequest (without `execute` method):
- https://docs.djangoproject.com/en/2.1/ref/request-response/#httprequest-objects
+https://docs.djangoproject.com/en/2.1/ref/request-response/#httprequest-objects
 """
 
-import os
 
+class Command:
+    """
+    The interface that commands implement. This is used to abstract invoker from
+    the command is going to handle the job.
+    """
+
+    def execute(self):
+        raise NotImplementedError()
 
-class MoveFileCommand:
-    def __init__(self, src, dest):
-        self.src = src
-        self.dest = dest
+
+class MakeBoldCommand(Command):
+    """
+    A simple command to bold a text.
+    """
 
     def execute(self):
```

Comment:
To show the effect of the command is good to have some definition of bold. Maybe in this case wrap the text with the HTML `<b>` or `<strong>` element. The same goes for italic.